src/main/scala/devices/spi/SPIPhysical.scala

```diff
@@ -59,11 +112,22 @@ class SPIPhysical(c: SPIParamsBase) extends Module {
     Mux(fmt.endian === SPIEndian.MSB, data, Cat(data.toBools))
 
   val rxd = Cat(io.port.dq.reverse.map(_.i))
-  val samples = Seq(rxd(1), rxd(1, 0), rxd)
+  val rxd_delayed = Vec(Seq.fill(io.port.dq.size)(false.B))
```

Comment:
Just realized this doesn't quite make sense. what is `rxd_delayed`, a Reg or Wire? You should wrap the `Vec` with either `Reg` or `Wire`.

Reply:
Couldn't get vector wire initiation without using this way of declaring in chisel2